variables.tf

```diff
@@ -14,6 +14,11 @@ variable "name" {
   description = "Name of security group"
 }
 
+variable "use_name_prefix" {
+  default     = "false"
+  description = "Use name_prefix parameter to set the Security Group name instead of the name parameter"
```

Comment:
```hcl
description = "Whether to use name_prefix or fixed name. Should be true to able to update security group name after initial creation"
default     = true
```

Keep default to `true` to allow users to update name and description as they want after the creation. Also, for backward-compatibility reason.